display_list/effects/dl_image_filter.h

```diff
@@ -7,6 +7,7 @@
 
 #include <utility>
 
+#include "display_list/effects/dl_color_source.h"
```

Comment:
Is this stale now?

Reply:
We need this for the samplers field on the DlRuntimeEffect filter